### PR TITLE
GAWB-1486: Save DUL option choice

### PR DESCRIPTION
**Pull request by MattPutnam**
Also, some wording changes.

Corresponding Orch PR: https://github.com/broadinstitute/firecloud-orchestration/pull/361

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/library/library_page.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/page/library/library_page.cljs:198]: thank you for getting rid of these console errors!

**Review by skwalker — COMMENTED** [review]
looks good to me, assuming moran likes it!

**Review by davidangb — COMMENTED** [review]
I find the orsp/questionnaire UX to be confusing - after you select an option, it's not clear to me as a user how to revert the choice, and I can't see what the other choice even is until I click through.

That said, I defer to our UI/UX folks. The code here looks good.

**Comment by MattPutnam** [comment]
Isaac agrees that with so much content disappearing and reappearing, it's a little disorienting.  A few quick animations may really help this.  Investigating now.

**Comment by skwalker** [comment]
👍 , once merge conflicts fixed

**Comment by davidangb** [comment]
👍 as TL

**Review by davidangb — APPROVED** [review]

**Review by davidangb — APPROVED** [review]
👍 as TL but don't merge until after today's snapshot!

Also, automated build failed, looks like it was a transient error, but please keep an eye on the build.

**Pull request merged**
